rpcclient/examples/bitcoincoreunixsocket/main.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2014-2017 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+
+	"github.com/btcsuite/btcd/rpcclient"
+)
+
+func main() {
+	// Connect to local bitcoin core RPC server using HTTP POST mode over a Unix Socket.
+	connCfg := &rpcclient.ConnConfig{
+		// For unix sockets, unix:// + "your unix socket path"
```

Comment:
nit: all comments should be full sentences with punctuation, here and the inline comments below.